hadoop-master1/files/opt/ssh/authorized_keys

```diff
@@ -0,0 +1,4 @@
+ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABgQCe/0PwjMh/eD8KmxyJ/azxql7HdZ3WVr4spkplMTzZbMyTdycZ0OUf+xqT1c+4YW+TpirG33n3AvCbUza24ivj17ypavUUvb+iWD54VgET77XKnG3GWy7Vkm/SxZI6t0W6sgrneYeBo39w452dl84j0dJwpDkmz0HPvcArsX7qn5uZcRgPg15sUz3rCthQ5a4146sLlkypxbpjWm8xvyayZDi0h8zoh57jIOprYLynFxgiuyQ28JSEEXbwsp94c+5Q90YXVgDj7HIqtlFPHFjjZEJXFnoDzCeGu1WzFAkHC2j4jxZ6xYP0CktuP04Y91KZtOvM/WwB+57Caxix/zjuFs9vC65L26ayrBbPjeMrFApS5cHovrY4732aCHjfFSy21HIL2+Z7KUmM15AtUkwH/+rBSfh05FGrQFO65O4qVMnQaZPG0l4EdrKVWarXNb9RRQzR06cT4+pGUmOMnvMcFoLhOgnSJZNERgDSX1FKsFPYr/1/vUJn/ScBNE4yHUs= root@hadoop-master1.orb.local
```

Comment:
one pair of the rsa key is sufficient, we can name it `root@hadoop-testing` instead of `root@hadoop-<balabala>.orb.local`